src/dev/build/tasks/clean_tasks.js

```diff
@@ -233,3 +233,21 @@ export const CleanExtraBrowsersTask = {
     }
   },
 };
+
+export const CleanEmptyFoldersTask = {
```

Comment:
Do we end up with empty directories from CleanExtraFilesFromModulesTask?

Reply:
Yes! In order to determine the files to being delete here https://github.com/elastic/kibana/pull/22618/files/6fcc285515023b26445f403364bbd7efb7446b8f#diff-522ca8ed6922cfa14cf9f7a32daa9338R80 we are using the `globby`. The expand functionality of globby sometimes will not return the folder containing the files itself so we will delete all the folder content but not the folder itself. IT's a behaviour documented on globby.